blocks/bricks/wrappers.py

```diff
@@ -34,3 +35,35 @@ def apply(self, input_):
         else:
             output = self.application_method(input_)
         return output
+
+
+class WithAxesSwapped(Brick):
+    """Wraps an application around dimshuffles.
```

Comment:
I'd say something more like "Swaps axes in both the input and output of an application."